site-src/implementations.md

```diff
@@ -241,10 +241,14 @@ FSM support of Gateway API is built on top [Flomesh Gateway API](fgw) and it cur
 ### Gloo Gateway
 
 [![Conformance](https://img.shields.io/badge/Gateway%20API%20Conformance%20v1.0.0-GlooGateway-green)](https://github.com/kubernetes-sigs/gateway-api/blob/main/conformance/reports/v1.0.0/gloo-gateway)
```

Comment:
Keeping 1.0.0 conformance status on purpose to show full compliant with 1.0.0 and partial with 1.1.0.